README.md

```diff
@@ -53,6 +53,9 @@ jekyll serve
 bundle exec jekyll serve
 ```
 
+**NOTE:** If the above serve command throws error saying `require': cannot load such file -- webrick (LoadError)` try to run `bundle add webrick` (automatically adds webrick gem to the Gemfile) or manually add `gem "webrick"` line to the Gemfile and then run the serve command again.
```

Comment:
This is a great tidbit! Here is a tiny tweak/suggestion:

```suggestion
**NOTE:** If the above serve command throws an error saying `require': cannot load such file -- webrick (LoadError)` try to run `bundle add webrick` to automatically add the webrick gem to your Gemfile, or manually add `gem "webrick"` line to the Gemfile and then run the serve command again.
```

Reply:
Sounds good. Done